builders/interoperability/xcm/xc-integration.md

```diff
@@ -85,50 +85,70 @@ When getting started with the Moonbase Alpha relay chain, once you have your sov
+
+1. Open (or ensure there is) an HRMP channel from your chain to Moonriver/Moonbeam. Optionally, register MOVR/GLMR
+2. Create [two Moonbeam Community forum posts](#forum-templates) with some key information for the XCM integration: 
+    - An [XCM Disclosure post](#xcm-disclosures), where you'll provide some disclosures about the project, the code base, and social network channels
+    - An [XCM Proposal post](#xcm-proposals), where you'll provide some techincal information about the proposal itself. 
+3. Create a batched proposal on Moonbeam/Moonriver to:
+    1. Accept the incoming HRMP channel 
+    2. Propose the opening of an outgoing HRMP channel from Moonriver/Moonbeam
     3. Register the asset as an [XC-20 token](/builders/interoperability/xcm/xc20/overview){target=_blank} (if applicable)
 
       The normal enactment times are as follows:  
 
       - **Moonriver** - proposals should be done in the the General Admin Track from [OpenGov](/learn/features/governance/#opengov){target=_blank}, in which the Decision Period is approximately {{ networks.moonriver.governance.tracks.general_admin.decision_period.time }}, and the enactment time is at least {{ networks.moonriver.governance.tracks.general_admin.min_enactment_period.time }}
       - **Moonbeam** - approximately a {{ networks.moonbeam.democracy.vote_period.days }}-day Voting Period plus {{ networks.moonbeam.democracy.enact_period.days }}-day enactment time
 
-4. Add details about the [connecting parachain on Polkassembly](https://moonbeam.polkassembly.network/){target=_blank} so that community members can be informed when voting on the proposal
-5. Accept the HRMP channel from Moonriver/Moonbeam on the connecting parachain
-6. Exchange $50 worth of tokens for testing the XCM integration. Please send the tokens to:
+4. Accept the HRMP channel from Moonriver/Moonbeam on the connecting parachain
+5. Exchange $50 worth of tokens for testing the XCM integration. Please send the tokens to:
 
     ```
     AccoundId: 5DnP2NuCTxfW4E9rJvzbt895sEsYRD7HC9QEgcqmNt7VWkD4
     Hex:       0x4c0524ef80ae843b694b225880e50a7a62a6b86f7fb2af3cecd893deea80b926)
     ```
 
-7. Provide an Ethereum-styled address for MOVR/GLMR
-8. Test the XCM integration with the provided tokens
+6. Provide an Ethereum-styled address for MOVR/GLMR
+7. Test the XCM integration with the provided tokens
 
 Once these steps are completed succesfully, marketing efforts can be coordinated, and the new XC-20 on Moonriver/Moonbeam can be added to the Cross Chain Assets section of the [Moonbeam DApp](https://apps.moonbeam.network/){target=_blank}.
 
-### Forum and Polkassembly Template {: #forum-and-polkassembly-template }
+## Forum Templates {: #forum-templates }
+
+When starting an XCM integration on Moonriver or Moonbeam MainNet, there are two preliminary posts that must be made on the Moonbeam Community Forum so that the voting community has the chance to provide feedback.
```

Comment:
```suggestion
When starting an XCM integration on Moonriver or Moonbeam MainNet, there are two preliminary posts that must be made on the [Moonbeam Community Forum](https://forum.moonbeam.foundation/){target=_blank} so that the voting community has the chance to provide feedback.
```